### ISSUE TITLE
Work with Namespace other the dcdr Info is alway empty

### ISSUE THREAD

**Issue by fruch** [vsco/dcdr#64]
I was trying Namespace other then dcdr (search in this case) and the dcdr.json looks like that:

```json
{
  "dcdr": {
    "info": {},
    "features": {
      "default": {
        "featue1": true,
        "featue2": 1,
        "feature-a": 0,
        "feature1": true
      },
      "search": {
        "info": null
      },
      "usr": {
        "12345": {
          "featue2": 0.5
        }
      }
    }
  }
```

**Comment by fruch** [comment]
found the issue and solved it in #65

**Comment by fruch** [comment]
fix is now merged
